src/bitcoin-cli.cpp

```diff
@@ -928,6 +938,46 @@ static void GetProgressBar(double progress, std::string& progress_bar)
     }
 }
 
+/**
+ * GetFeerateBar constructs fee rate distribution bars using dynamic increment based on size.
+ *
+ * @param[in]  size      Size of fee rate group in bytes.
```

Comment:
Maybe the arguments can be re-ordered so that `in` params are first and the `out` param is last.

Reply:
Done